### ISSUE TITLE
what is the config file used for pretrained model pp_model_for_nuscenes_pretrain/voxelnet-296960.tckpt?

### ISSUE THREAD

**Issue by chowkamlee81** [traveller59/second.pytorch#198]
Hi,

The pretrained model pp_model_for_nuscenes_pretrain/voxelnet-296960.tckpt, downloaded.

What is the configuration file used to develop this? Kindly help

**Comment by cvnerds** [comment]
maybe configs/pointpillars/pp_pretrain.config

**Comment by zovathio** [comment]
could you find out? for maybe configs/pointpillars/pp_pretrain.config I got a dimension mismatch error

**Comment by HanFa** [comment]
I also get the shape mismatch errors for RPN with config file `configs/poiintpillars/pp_pretrain.config` and all config files in folder `configs/nuscenes`. Could anyone provide the proper config?

**Comment by yinjunbo** [comment]
I'd like to know which config file yield the best results on nuScenes. Does anyone figure it out?

**Comment by lawrencekiba** [comment]
I have this dimension mismatch issue as well.

**Comment by AhmedMagdyHendawy** [comment]
Anyone knows which config file?

**Comment by ICS-India** [comment]
@AhmedMagdyHendawy @chowkamlee81 @traveller59 

in all.pp.largea.config/all.pp.mida.config/all.pp.lowa.config  if you do the following modification you will be able to load the network  
1) In RPN change the upsample stride  to this  "upsample_strides : [1, 2, 4]"
2) In target assigner keep only one class ("I kept car alone ")

Then you should be able to load the network and evaluate the samples . But i got zero accuracy with NUSCENES mini dataset . Let me know if anybody able to get reasonable accuracy this pretrained network . original config may be different  ?

**Comment by yuanzheng-s** [comment]
@ICS-India hi, i follow the way of your modification,and build_network successful!   but the inference result in the kittiviewer web is strange, the predict bbox have large difference with the ground truth,have you met this before
